src/tests/fixtures/form.html

```diff
@@ -8,7 +8,17 @@
     <script type="module">
       import dialogPolyfill from "https://cdn.skypack.dev/dialog-polyfill"
 
-      window.dialogPolyfill = dialogPolyfill
+     addEventListener("click", ({ target }) => {
```

Comment:
@sstephenson I was unable to bridge the gap between loading the dialog polyfill and executing arbitrary JS from the test TypeScript file.

I think this will pass the suite, but I'm open to suggestions on how to dynamically pull in the polyfill and apply it from TS.